demo/src/style/app.scss

```diff
@@ -101,10 +106,54 @@ p.github-buttons > iframe {
   background-color: transparent
 }
 
-div.component-api,
-div.component-api div.component-api-property,
-div.component-api div.component-api-instructions {
-  margin-bottom: 10px;
+div.api-doc-component {
+  margin-bottom: 20px;
+}
+
+div.api-doc-component > h2 > a {
+  text-decoration: none;
+}
+
+div.api-doc-component > h2 > a:hover {
+  text-decoration: none;
+}
+
+div.api-doc-component > h2 {
+  margin-top: 0.2em;
+  margin-bottom: 0.2em;
+  line-height:0.85em;
+  text-shadow:
+          0 0 3px white, /* stroke around*/
```

Comment:
The initial underlining of the hyperlinks was horrendous using the default behavior of the browser.  The hack of using the border bottom was significantly better, but was too far from the text, especially with lowercase letters dropping below the line.  A bunch of searching yielded this solution which I neglected to reformat.  Basically, though, it splits the underline when you have lowercase letters that drop below the line.